README.md

```diff
@@ -84,6 +84,27 @@ mbedConnectorApi.startLongPolling(function(error) {
 });
 ```
 
+### Setup callback notifications
+```javascript
+var MbedConnectorApi = require('mbed-connector-api');
+var mbedConnectorApi = new MbedConnectorApi({
+  accessKey: /* Access Key */
+});
+
+// Replace `webappPutRouteCallback` with your web app's PUT request handler
+webappPutRouteCallback('/webhook', function (req, res) {
```

Comment:
I agree, thanks for the feedback @janjongboom. I've changed the example to use Express since the code is shorter and easier to understand, let me know what you think!